packages/config/test/index.test.js

```diff
@@ -211,105 +211,161 @@ test('should look for a .env file in process.cwd() too', async t => {
 
 test('ConfigManager.listConfigFiles() lists possible configs by type', async t => {
   assert.deepEqual(ConfigManager.listConfigFiles('db'), [
-    'platformatic.db.json',
-    'platformatic.db.json5',
-    'platformatic.db.yaml',
-    'platformatic.db.yml',
-    'platformatic.db.toml',
-    'platformatic.db.tml',
-    'platformatic.json',
-    'platformatic.json5',
-    'platformatic.yaml',
-    'platformatic.yml',
-    'platformatic.toml',
-    'platformatic.tml',
     'watt.json',
     'watt.json5',
     'watt.yaml',
     'watt.yml',
     'watt.toml',
     'watt.tml',
-  ])
-  assert.deepEqual(ConfigManager.listConfigFiles('service'), [
-    'platformatic.service.json',
-    'platformatic.service.json5',
-    'platformatic.service.yaml',
-    'platformatic.service.yml',
-    'platformatic.service.toml',
-    'platformatic.service.tml',
     'platformatic.json',
     'platformatic.json5',
     'platformatic.yaml',
     'platformatic.yml',
     'platformatic.toml',
     'platformatic.tml',
+    'watt.db.json',
+    'watt.db.json5',
+    'watt.db.yaml',
+    'watt.db.yml',
+    'watt.db.toml',
+    'watt.db.tml',
```

Comment:
These should _not_ be supported, we should be dropping the `.service` and `.db`, etc.

Reply:
no. We still need to support the legacy `platformatic.db.json` like we were before. But we are going to remove that in v3.

Reply:
Removed!